doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib

```diff
@@ -434,7 +434,7 @@ The following strings must not be used as metadata names:
 ## Ordering Results
 
 Operations that retrieve lists of entities permit the `orderBy` URI parameter to specify the
-attributes or properties to be be used as criteria when ordering results.
+attributes or properties to be used as criteria when ordering results.
```

Comment:
fiware-ngsiv2-2.0-2018_09_15.apib is frozen as v2.0.0 release of the NGSIv2. Thus, I'd prefer not touching it, even for typo fixing.

Could you do a copy of the file naming the new one as fiware-ngsiv2-2.0.1-wip.apib, please? No problem if the PR shows an enteritly new file for it, I'll check it with diff command before merging.

Thanks!